docs/DeveloperGuide.adoc

```diff
@@ -761,41 +775,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |user |find my link:#frequently-used-contacts[frequently used contacts] at the top of the displayed list when I do a search |quickly find the contact information I need
 
-|`* *` |user |see frequently used contacts instantly |locate them more easily
+|`* *` |regular user|see frequently used contacts instantly |locate them more easily
```

Comment:
For consistency purposes, put another space before the "|"
Like "|regular user|" becomes "|regular user |"

Same applies for line 784.